cluster-up/cluster/ephemeral-provider-common.sh

```diff
@@ -77,6 +77,22 @@ function _add_common_params() {
         params=" --enable-ceph $params"
     fi
 
+    if [[ $KUBEVIRT_DEPLOY_PROMETHEUS == "true" ]] && 
```

Comment:
Not sure, maybe we should check what provider is used here also, and fail if it doesn't support prometheus, like i.e. >= 1.21?

Reply:
I think it makes sense!
Not sure what is better: fail or echo a warn?